src/main/java/com/example/onlinebookstore/dto/ArgumentNotValidResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.example.onlinebookstore.dto;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+@Data
+public class ArgumentNotValidResponse {
```

Comment:
I think you can convert it to general exception response format and use this class for all handlers